_posts/2022-11-22-redis-job-queue-reloaded.adoc

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: 'Redis Job Queue - Reloaded'
+date: 2022-11-22
+tags: redis
+synopsis: 'How to implement a more reliable job queue with Redis, including fault tolerance, de-duplication, and poison pill.'
+author: cescoffier
+---
+
+In https://quarkus.io/blog/redis-job-queue/[How to implement a job queue with Redis], we explained how to implement a job queue mechanism with Redis and the new Redis API from Quarkus. 
+The approach explored in that blog post had a significant flaw: if the execution of a job failed, the request was lost and will never be re-attempted.
+
+In this post, we explain how to improve the reliability of the job queue to handle failures, enable retry and use a _dead-letter queue_ to avoid poison pills. 
+
+## Recap & Problem
+
+In the https://quarkus.io/blog/redis-job-queue/[previous blog post], we implemented the following system.
+
+image::/assets/images/posts/redis-job-queue/application.png[align="center"]
+
+An application receives _fight requests_ and writes these requests into a Redis List. 
```

Comment:
Do you want an uppercase `L` for `Redis List`? I see that below, you write it `list`.

Reply:
changed to lowercase.